README.md

```diff
@@ -27,7 +27,32 @@ Download the binary for your platform from the [releases](https://github.com/ran
 wget https://github.com/rancher/terraform-provider-rke/releases/download/x.x.x/terraform-provider-rke_<platform>
 ```
 
-Rename the binary to to remove the platform ending (so it looks like `terraform-provider-rke`)and move it to your terraform plugin directory (`~/terraform.d/plugins/linux_amd64`) or a custom directory. If using a custom directory use the `-plug-in=/my/custom/path` when running `terraform init` to specify the path.
+1. rename it to `terraform-provider-rke_v0.14.1`
```

Comment:
Binary should be renamed to `terraform-provider-rke_<VERSION>` being VERSION in the format vX.Y.Z  . I think it's better to let version as variable.

Reply:
I think line 27 is not clear also, let me change both of it

Reply:
Ok, but please don't specify any concrete version. In fact, current and updated version is v1.0.0-rc5 that supports RKE v1.1.0

Reply:
updated thx